src/common/config.h

```diff
@@ -162,6 +162,7 @@ struct md_config_t {
   int _get_val(const std::string &key, char **buf, int len) const;
   Option::value_t get_val_generic(const std::string &key) const;
   template<typename T> T get_val(const std::string &key) const;
+  template<typename T> T _get_val(const std::string &key) const;
```

Comment:
do we have a caller of this function now?

Reply:
<pre>
+    if (changed.find("log_coarse_timestamps") != changed.end()) {
 +      log->set_coarse_timestamps(conf->_get_val<bool>("log_coarse_timestamps"));
 +    }
</pre>